numba/parfors/parfor_lowering.py

```diff
@@ -937,6 +1005,51 @@ def _create_gufunc_for_parfor_body(
     gufunc_txt += "def " + gufunc_name + \
         "(sched, " + (", ".join(parfor_params)) + "):\n"
 
+    globls = {"np": np}
+
+    # First thing in the gufunc, we reconstruct tuples from their
+    # individual parts, e.g., orig_tup_name = (part1, part2,).
+    # The rest of the code of the function will use the original tuple name.
+    for tup_var, exp_names in tuple_var_to_expanded_names.items():
+        tup_type = typemap[tup_var]
+        gufunc_txt += "    " + param_dict[tup_var]
+        # Determine if the tuple is a named tuple.
+        if (isinstance(tup_type, types.NamedTuple) or
+            isinstance(tup_type, types.NamedUniTuple)):
+            named_tup = True
+        else:
+            named_tup = False
+
+        if named_tup:
+            # It is a named tuple so try to find the global that defines the
+            # named tuple.
+            found = False
+            func_def = guard(get_definition, lowerer.func_ir, tup_var)
+            if config.DEBUG_ARRAY_OPT:
+                print("func_def:", func_def, type(func_def))
+            if (func_def is not None and
+                isinstance(func_def, ir.Expr) and
+                func_def.op == 'call'):
+                func_def = guard(get_definition, lowerer.func_ir, func_def.func)
+                if config.DEBUG_ARRAY_OPT:
+                    print("func definition:", func_def, type(func_def))
+                if func_def is not None:
+                    if isinstance(func_def, ir.Global) or isinstance(func_def, ir.FreeVar):
```

Comment:
```suggestion
                    if isinstance(func_def, (ir.Global ir.FreeVar)):
```
`isinstance` can take a tuple of types as the second arg.

Reply:
I always forget that, thanks.